src/MTGCore/ViewModels/DeckViewModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using MTGCore.Dtos.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MTGCore.ViewModels
+{
+    public class DeckViewModel
+    {
+        public List<CardAmt> Cards { get; set; }
+    }
+
+    public class CardAmt
+    {
+        public CardDto Card { get; set; }
+        public int Amount { get; set; }
+    }
```

Comment:
I like this solution. I assume it is so you only need one of each CardDto regardless of how many of that card is in the deck. Seems like it would be more efficient than storing multiple of the same CardDto in a deck.

Reply:
Exactly that. The view only needs to display the card name and the amount in each deck. Eg.

- Shock x2